js/gui.js

```diff
@@ -3,23 +3,14 @@
 // defined in /js/levels.js
 /* global createTestLevel */
 var player;
-var level;
-
 
 window.addEventListener("load", function() {
   startGame();
 });
 
-function clearGame() {
-  $("#tiles").html("");
-}
-
 function startGame() {
-  var container = document.getElementById("tiles");
-  level = createFirstLevel();
-  level.showIn(container);
   player = newPlayer();
-  player.enterLevel(level);
+  player.enterLevel(createFirstLevel());
```

Comment:
should we manually specify this? `createFirstLevel()`

Reply:
that is in levels.js

Reply:
I know that I mentoned whether we should isolate functions as such?

Reply:
I think, it may make merging easier. Is this a blocker for merging? I am curiuos, as I would like to merge this and have the other levels added.

Reply:
also: highest form is to have a descriptive language, see levels.js if we split it from level.js, the implementation. this way we have to understand less to make a change

Reply:
>  I am curiuos, as I would like to merge this and have the other levels added.
> also: highest form is to have a descriptive language, see levels.js if we split it from level.js, the implementation. this way we have to understand less to make a change

Yes, carry on this is no blocker to merger. I was just putting up a suggestion, seems like your idea is much more better.